pkg/asset/rhcos/bootstrap_image.go

```diff
@@ -37,6 +39,15 @@ func (i *BootstrapImage) Generate(p asset.Parents) error {
 	p.Get(ic)
 	config := ic.Config
 
+	// Check for RHCOS image URL override
+	if config.Platform.Name() == baremetal.Name {
+		if boi := config.Platform.BareMetal.BootstrapMachineOSImage; boi != "" {
+			logrus.Infof("Found an URL override in Baremetal Platform for Bootstrap Machine OS Image: %s", boi)
+			*i = BootstrapImage(boi)
+			return nil
+		}
+	}
```

Comment:
I don't have a strong opinion about this, but I think it might be clearer if this was in the baremetal case below

Reply:
+1. This makes a ton of sense. Will update.